project/SparkBuild.scala

```diff
@@ -273,10 +273,9 @@ object SparkBuild extends PomBuild {
       // Google Mirror of Maven Central, placed first so that it's used instead of flaky Maven Central.
       // See https://storage-download.googleapis.com/maven-central/index.html for more info.
       "gcs-maven-central-mirror" at "https://maven-central.storage-download.googleapis.com/maven2/",
-      DefaultMavenRepository,
-      Resolver.mavenLocal,
-      Resolver.file("ivyLocal", file(Path.userHome.absolutePath + "/.ivy2/local"))(Resolver.ivyStylePatterns)
-    ),
+      DefaultMavenRepository) ++
+      { if (sys.env.contains("SKIP_LOCAL_M2")) Nil else Seq(Resolver.mavenLocal) } :+
```

Comment:
hmm... so, even if the value of `SKIP_LOCAL_M2` is "", `Resolver.mavenLocal` will be disabled?

Reply:
Let's make the conditions more stricter.
Updated.